samples/visual_indicator/nodejs/README.md

```diff
@@ -0,0 +1,129 @@
+---
+page_type: sample
+description: Microsoft Teams meeting extensibility sample for iteracting with In-meeting notifications
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+extensions:
+ contentType: samples
+ createdDate: "23/05/2023 13:38:26 PM"
+urlFragment: officedev-microsoft-teams-samples-visual-indicator-nodejs
+---
+
+## Targeted In-Meeting Notification
+
+This sample illustrates how to send visual indicator notification using [Targeted In-Meeting Notification](https://learn.microsoft.com/microsoftteams/platform/apps-in-teams-meetings/meeting-apps-apis?branch=pr-en-us-7615&tabs=dotnet#targeted-meeting-notification-api)  In-Meeting Experience.
+
+## Included Features
+* Bots
+* In-Meeting Notifications
+* Adaptive Cards
```

Comment:
If it's a simple feature that can be integrated in existing sample, why are we creating new sample?